### PR TITLE
Toasts

### PR DESCRIPTION
**Pull request by iammatis**

### CODE REVIEW

File: cryptoinvestor/views/assets.py [hunk 1]
```diff
@@ -42,10 +42,36 @@ def get_objects(self):
...
+                message = f"From last time {id_} went down by {value}!"
+                color = 'red'
+            else:
+                message = f"From last time {value} did not change!"
+                color = 'blue'
+
+            toasts.append({
+                'message': message,
+                'color': color
+            })
```
Comment (mskrip) [inline cryptoinvestor/views/assets.py]: Please export this to a separate function whose sole purpose is to generate toast messages from assets. This function is getting too complex.

File: cryptoinvestor/views/assets.py [hunk 2]
```diff
@@ -42,10 +42,33 @@ def get_objects(self):
...
+                message = f"From last time {id_} went up by {value}!"
+                color = 'green'
+            elif value < 0:
+                message = f"From last time {id_} went down by {value}!"
+                color = 'red'
+            else:
+                message = f"From last time {value} did not change!"
+                color = 'blue'
+
+            toasts.append(self.app.add_toast(message, color))
```
Comment (mskrip) [inline cryptoinvestor/views/assets.py]: This is good but I meant for you to export this whole block (from 46 to 65 lines)

Comment (iammatis) [reply cryptoinvestor/views/assets.py]: Should we make two different add_toast functions ? One for general toasts with simple message and another for more complex ones such as this one, when we want to differentiate the colors and texts ?

Comment (mskrip) [reply cryptoinvestor/views/assets.py]: It is good to have general add_toast method in app. But yes there should be another asset-reporting specific, at least a helper function, not necessarily assigned to a class.

File: cryptoinvestor/static/js/toasts.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+function  makeToasts(array) {
+    console.log(array);
```
Comment (mskrip) [inline cryptoinvestor/static/js/toasts.js]: This logging should not be pushed into production

File: cryptoinvestor/static/js/toasts.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+function  makeToasts(array) {
```
Comment (mskrip) [inline cryptoinvestor/static/js/toasts.js]: I get this error message in browser console on index page:

```
Uncaught TypeError: Cannot read property 'length' of undefined
    at makeToasts (toasts.js:3)
    at (index):55
```

**Comment by mskrip** [comment]
Would it be possible to move the toasts to the bottom of the page? Because now they may obstruct a view on some currencies on smaller screens

**Pull request merged**